templates/base/navigation.html

```diff
@@ -70,12 +70,12 @@
             </a>
           </li>
           <li class="moj-primary-navigation__item">
-            <a class="moj-primary-navigation__link" href="https://user-guide.find-moj-data.service.justice.gov.uk" rel="noreferrer noopener" target="_blank">
+            <a class="moj-primary-navigation__link" href="https://user-guide.find-moj-data.service.justice.gov.uk">
               {% translate "User guide" %}
             </a>
           </li>
           <li class="moj-primary-navigation__item">
-            <a class="moj-primary-navigation__link" href="https://user-guide.find-moj-data.service.justice.gov.uk/#contact-us" rel="noreferrer noopener" target="_blank">
+            <a class="moj-primary-navigation__link" href="https://user-guide.find-moj-data.service.justice.gov.uk#contact-us">
```

Comment:
```suggestion
            <a class="moj-primary-navigation__link" href="https://user-guide.find-moj-data.service.justice.gov.uk/#contact-us">
```
Link works, but opens with the `/`